test/src/org/labkey/test/tests/targetedms/TargetedMSDocumentFormatsTest.java

```diff
@@ -14,39 +14,121 @@
  */
 package org.labkey.test.tests.targetedms;
 
+import org.junit.Assert;
+import org.junit.BeforeClass;
 import org.junit.Test;
 import org.junit.experimental.categories.Category;
-import org.labkey.test.BaseWebDriverTest;
+import org.labkey.remoteapi.CommandException;
+import org.labkey.remoteapi.Connection;
+import org.labkey.remoteapi.query.Filter;
+import org.labkey.remoteapi.query.Row;
+import org.labkey.remoteapi.query.SelectRowsCommand;
+import org.labkey.remoteapi.query.SelectRowsResponse;
+import org.labkey.remoteapi.query.Sort;
 import org.labkey.test.categories.DailyB;
 import org.labkey.test.categories.MS2;
 
+import java.io.IOException;
 import java.util.Arrays;
-import java.util.List;
+import java.util.Collections;
 
 @Category({DailyB.class, MS2.class})
-@BaseWebDriverTest.ClassTimeout(minutes = 6)
 public class TargetedMSDocumentFormatsTest extends TargetedMSTest
 {
     private static final String SAMPLEDATA_FOLDER = "DocumentFormats/";
 
-    public static final List<String> skyZipFileNames = Arrays.asList(
-            "DocumentSerializerTest_36",
-            "DocumentSerializerTest",
-            "DocumentSerializerTest_compact",
-            "CrosslinkingExample",
-            "CrosslinkingExample_compact");
+    private static int JOB_COUNT;
+
+    @BeforeClass
+    public static void setupProject()
+    {
+        TargetedMSDocumentFormatsTest init = (TargetedMSDocumentFormatsTest) getCurrentTest();
+        init.setupFolder(FolderType.Experiment);
+    }
 
     @Test
-    public void testDocumentSerialization()
+    public void test36Document()
     {
-        setupFolder(FolderType.Experiment);
-        for (String filename : skyZipFileNames) {
-            testImportDoc(filename);
-        }
+        goToProjectHome(getProjectName());
+        importData(SAMPLEDATA_FOLDER + "DocumentSerializerTest_36.sky.zip", ++JOB_COUNT);
+    }
+
+    @Test
+    public void testNormalDocument()
+    {
+        goToProjectHome(getProjectName());
+        importData(SAMPLEDATA_FOLDER + "DocumentSerializerTest.sky.zip", ++JOB_COUNT);
```

Comment:
No verification needed?

Reply:
That's right. The original version of this test just imported three files and verified there were no errors. This branch has added two new files with cross-linked peptide data, and those are the ones where I did more extensive validation.